x12-edi/2.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,593 @@
+
+=== To Define your Implementation Convention with an Overlay Schema
+
+To specify a schema according to your implementation convention, you can follow this process:
+
+. Create an overlay schema that imports the base schema you want to customize - for example, X12 005010 850.
+. Customize the overall structure for segment usage, positions, groups, and counts.
+. Customize segments, including usage and counts.
+
+Overlay schemas are very similar in structure to a xref:1.0@partner-manager::edi-schema-language-reference.adoc[complete schema definition], but instead of providing all the details of the schema structure they only list changes. Overlay schemas specify how to use implementation conventions with a particular trading partner to extend and customizes the standard.
+
+The following example provides the start of a sample overlay schema that modifies the basic X12 005010 850 transaction set definition. This example customizes the CUR segment and specifies that it is unused, thereby ensuring it is hidden from the mapping structures in Anypoint Studio.
+
+[source,yaml,linenums]
+----
+form: X12
+version: '005010'
+imports: [ '/x12/005010/850.esl' ]
+structures:
+- idRef: '850'
+  name: Purchase Order
+  class: PO
+  heading:
+  - { idRef: 'CUR', position: '0400', usage: U }
+----
+
+=== Structure Overlay
+
+A structure overlay details modifications to the base schema definition of an X12 transaction set. Most often these modifications take the form of marking segments or groups in the base definition as unused, but any usage or repetition count change is allowed.
+
+The following is the form taken by a structure overlay:
+
+[source,yaml,linenums]
+----
+- idRef: '850'
+  heading:
+  - { idRef: 'CUR', position: '0400', usage: U }
+  - { idRef: 'PER', position: '0600', usage: U }
+  - groupIdRef: 'N9_Loop'
+    position: '2950'
+    items:
+    - { idRef: 'DTM', position: '2970', usage: U }
+    - { idRef: 'PWK', position: '3050', usage: U }
+    - { idRef: 'EFI', position: '3080', usage: U }
+----
+
+The modifications in this example specify that the CUR and PER segments of the standard 850 heading are not being used, along with the segments in the N9 loop.
+
+The key-value pairs at the structure level are:
+
+[%header,cols="30a,70a"]
+|===
+|Key |Description
+|idRef |The ID for the transaction set being modified.
+|name |The transaction set name (optional).
+|heading, detail, summary |List of segment and group modifications within each section of the structure. These are optional, and each is only used when there are modifications to that section.
+|===
+
+The lists of segment modifications for the different sections of the structure (heading, detail, summary) all use the same structure. Each item in the list is either a segment reference or a group definition. Segment references are shown using a compact YAML syntax where the values for each reference are given as comma-separated key-value pairs enclosed in curly braces. The values are:
```

Comment:
"using a compact YAML syntax where" > "using a compact YAML syntax in which"

Reply:
fixed